docs/reserved-keywords-spec-version-2019.adoc

```diff
@@ -0,0 +1,294 @@
+= PartiQL Reserved Keywords
+
+The PartiQL language uses reserved keywords to parse and understand PartiQL statements.
+PartiQL Language Grammar Reserved Keywords cannot be used in PartiQL statements as-is.
```

Comment:
"Reserved Keywords" -- no need to capitalize, I'd think (the previous line doesn't). 

"cannot be used in PartiQL statements as-is" --> "cannot be used as non-delimited identifiers".  That's what a keyword means commonly. "Delimited" rather than "quoted" is the SQL terminology, I believe, but that's secondary. 

Style-wise, 4 occurrences of "PartiQL" in 2 lines is a bit much. :-)

Reply:
> Style-wise, 4 occurrences of "PartiQL" in 2 lines is a bit much. :-)
Yep, tbh it was intentional to add some SEO twist to it as this will be in the website :)
Let me see if I can tone it down a bit.

Reply:
Oh, with SEO in the picture, I wash my hands. :-)